koku/masu/database/report_manifest_db_accessor.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2018 Red Hat, Inc.
+# Copyright 2020 Red Hat, Inc.
```

Comment:
```suggestion
# Copyright 2018 Red Hat, Inc.
```

Reply:
We only change these for when the file was created.  @blentz can keep me honest here.